----------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~\\
  |\\ | |__  __ /  ` /  \\ |__) |__         }  {
  | \\| |       \\__, \\__/ |  \\ |___     \\`-._,-`-,
                                        `._,._,'
  nf-core/testworkflow v${version}
----------------------------------------------------

Run Name: $runName

<% if (success){
    out << "## nf-core/testworkflow execution completed successfully! ##"
} else {
    out << """####################################################
## nf-core/testworkflow execution completed unsuccessfully! ##
####################################################
The exit status of the task that caused the workflow execution to fail was: $exitStatus.
The full error message was:

${errorReport}
"""
} %>


The workflow was completed at $dateComplete (duration: $duration)

The command used to launch the workflow was as follows:

  $commandLine



Pipeline Configuration:
-----------------------
<% out << summary.collect{ k,v -> " - $k: $v" }.join("\n") %>

--
nf-core/testworkflow
https://github.com/nf-core/testworkflow
